controllers/pod_models.go

```diff
@@ -518,6 +518,21 @@ func configureSidecarContainer(container *corev1.Container, initMode bool, insta
 		}
 
 		sidecarEnv = append(sidecarEnv, corev1.EnvVar{Name: "FDB_INSTANCE_ID", Value: instanceID})
+
+		if !initMode && cluster.Spec.SidecarContainer.EnableLivenessProbe && container.LivenessProbe == nil {
+			// We can't use a HTTP handler here since the server
+			// requires a client certificate
+			container.LivenessProbe = &corev1.Probe{
```

Comment:
Should we capture a follow-up task to remove the readiness probe, since it's not doing a lot of good for us right now?